src/Compilers/CSharp/Test/Syntax/Parsing/AsyncParsingTests.cs

```diff
@@ -1681,21 +1681,15 @@ class C
 {
     async partial implicit operator
 ",
-                // (4,11): error CS1553: Declaration is not valid; use '+ operator <dest-type> (...' instead
+                // (4,11): error CS1553: Declaration is not valid; use 'implicit operator <dest-type> (...' instead
```

Comment:
this is MUCH better.  the user wrote `implicit operator`.  we should be treating htat as a conversion operator with good messages on how to write it.